CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 #### 1.9.8-SNAPSHOT
+- **[FEATURE]** Added `ImmutableVector2`, an immutable equivalent to `Vector2`.
```

Comment:
Add the module name as well so that it's obvious where the class was added; see the previous feature logs.